client/src/game/Game.js

```diff
@@ -9,9 +14,34 @@ function resizeCanvas(canvas) {
   canvas.height = 667;
 }
 
+const directionKeyCodes = {
+  ArrowRight: DIRECTIONS.RIGHT,
+  ArrowLeft: DIRECTIONS.LEFT,
+  Space: DIRECTIONS.JUMPING,
+};
+
 function handleKeyDown(event) {
-  movePlayer(event.code, player);
-  rotatePlayer(event.code, player, DIRECTIONS);
+  keyState[event.code] = true;
+  const direction = directionKeyCodes[event.code];
+  if (direction) {
+    rotatePlayer(player, direction);
+  }
+
+  if (keyState[DIRECTIONKEYS.UP] && player.isAbleToJump & !player.isJumping) {
+    player.isJumping = true;
+    player.isAbleToJump = false;
+    setTimeout(() => {
+      player.isJumping = false;
+    }, 300);
+  }
+}
+
+function handleKeyUp(event) {
+  keyState[event.code] = false;
```

Comment:
You remember all keys (like `n`, `m`). 
Instead, you could check if the direction exists (if it is a used key).